Tasks/compare.js

```diff
@@ -1,19 +1,23 @@
 // Compare two dictionaries
+'use strict';
 
-let compare=(
-  (first_values,...parameters_LIST) => {
-   const second_values = parameters_LIST[0]
-   let a = Object.keys(first_values)
-   let b = Object.keys(second_values)
-   if (a.join('-') !== b.join('-')) return false;
-   let e = true;
-   for (c of a) {
-    if (first_values[c] === second_values[c]) e = e && true
-   else { e = e && false }
-   }
-  return e;
-  }
-)
+const compareDictionaries = function ( obj1, obj2 ) {
+    const obj1Keys = Object.keys(obj1).sort();
+    const obj2Keys = Object.keys(obj2).sort();
+    let e = true;
+    
+    if ( obj1Keys.length !== obj2Keys.length ) return false;
```

Comment:
Use eslint and prettier to check style against common conventions
```suggestion
    if (obj1Keys.length !== obj2Keys.length) return false;
```